### ISSUE TITLE
Cant use it ;)

### ISSUE THREAD

**Issue by JaniszM** [epoberezkin/sacjs#1]
I think there is missing index.js or in package.json in main section should be ./lib/cache.js
In current shape after npm install sacjs I cannot use it, node throws

```
Error: Cannot find module 'sacjs'
    at Function.Module._resolveFilename (module.js:326:15)
...
```

:(

**Comment by epoberezkin** [comment]
main should be "lib/cache.js" indeed

**Comment by epoberezkin** [comment]
Without the fix you can `require('sacjs/lib/cache')`

**Comment by JaniszM** [comment]
Yes. Will you do a quick fix? :)

**Comment by epoberezkin** [comment]
Fixed in 1.1.1, thank you :)

**Comment by JaniszM** [comment]
Thanks :)
